### PR TITLE
fix markdown and nextflow_schema.json check

### PR DESCRIPTION
**Pull request by yuukiiwa**
hi @cying111 , I have updated usage.md with more specific headers and added the missing 34 parameters to nextflow_schema.json to avoid errors.

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ As shown in the examples below, the accepted format of the file is slightly diff
 
 #### With basecalling and demultiplexing
 
-##### Example `samplesheet.csv`
+##### Example `samplesheet.csv` for barcoded fast5 inputs
```
Comment (cying111) [inline docs/usage.md:33]: any reason why you add this? Same for below

Comment (yuukiiwa) [reply docs/usage.md:33]: I got the following error from the markdownlint check:
Screenshot 2020-08-14 at 4 40 18 PM

Comment (yuukiiwa) [reply docs/usage.md:33]: there's this no duplicate heading requirement

**Comment by yuukiiwa** [comment]
hi @cying111, I made some changes and ran through the test profiles locally with the updated test-datasets:
test,docker
test
test_bc_nodx,docker
test_bc_nodx
test_nobc_dx,docker
test_nobc_dx
test_nobc_nodx,docker
test_nobc_nodx
test_nobc_nodx_noaln
test_nobc_nodx_noaln

**Comment by yuukiiwa** [comment]
hi @cying111, I updated the test-dataset samplesheets and all tests ran through:
test,docker
test
test_bc_nodx,docker
test_bc_nodx
test_nobc_dx,docker
test_nobc_dx
test_nobc_nodx,docker
test_nobc_nodx
test_nobc_nodx_noaln,docker
test_nobc_nodx_noaln

**Comment by yuukiiwa** [comment]
the tests ran through locally with the updated samplesheet format:
test,docker
test
test_bc_nodx,docker
test_bc_nodx
test_nobc_dx,docker
test_nobc_dx
test_nobc_nodx,docker
test_nobc_nodx
test_nobc_nodx_noaln,docker
test_nobc_nodx_noaln

**Pull request merged**